src/d3/scatterplot.d3.ts

```diff
@@ -44,14 +45,34 @@ export class Scatterplot{
     container.appendChild(this.dataPointContainer);
     container.appendChild(this.dataTextContainer);
     if (this.data.length > 0 && !this.loaded){
-      console.log(data);
-      this.generatePts();
       this.generateText();
+      this.generatePts();
       this.setColor('blue');
     }
     this.createSky('gray');
     this.createGridPlane();
     this.loaded = true;
+    window.onload = () => {
+      document.addEventListener('keydown', (event) => {
+        if (event.keyCode === 81){
```

Comment:
I think keyCode is deprecated: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Can use `code`: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code